app/serializers/user_notification_total_serializer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class UserNotificationTotalSerializer < ApplicationSerializer
+  attributes :id,
+             :username,
+             :unread_notifications,
+             :unread_personal_messages,
+             :unseen_reviewables,
+             :topic_tracking,
+             :group_inboxes
+
+  def unread_notifications
+    object.all_unread_notifications_count - object.new_personal_messages_notifications_count
+  end
+
+  def unread_personal_messages
+    object.new_personal_messages_notifications_count
+  end
+
+  def unseen_reviewables
+    Reviewable.unseen_reviewable_count(object)
+  end
+
+  def topic_tracking
+    TopicTrackingState.report_totals(object)
+  end
+
+  def include_group_inboxes?
+    scope.user.staff?
```

Comment:
I'm curious why this is only included for staff instead of any user with access to a group which has group messages.

Reply:
It's what we do in the mobile app, but you're right, no reason it needs to be limited to staff.